coding/internal/file_data.cpp

```diff
@@ -77,6 +78,7 @@ string FileData::GetErrorProlog() const
   case OP_WRITE_TRUNCATE: s = "Write truncate"; break;
   case OP_WRITE_EXISTING: s = "Write existing"; break;
   case OP_APPEND: s = "Append"; break;
+  default: UNREACHABLE();
```

Comment:
This should be unnecessary if `enum Op` will be a class enum. It was created before migrating to C++11 :)

Reply:
>This should be unnecessary if enum Op will be a class enum.

Negative. That does not help. Regardless if it is an eyesore I can refactor the code to code `enum class` in a different commit, but that does not help with the warning.

Reply:
Wow, interesting. Which compiler is it?

Reply:
Emits the warning: `gcc`  ,`12.3.0`,  `13.2.0`, `14.0.1` 
Doesn't emit the warning: `gcc` `11.4.0`